js/main.js

```diff
@@ -0,0 +1,26 @@
+// 1. Select element (using CSS selectors)
+var myButton = document.querySelector('.jumbotron');
+
+// 2. Define your event handler (callback <-- function)
+myButton.addEventListener('click', doSomething);
+
+function doSomething(){
```

Comment:
You could rename this function to something describing what it does.

Maybe 'changePageElementColours' or something :P